util/include/DigestImpl.ipp

```diff
@@ -97,7 +116,8 @@ class DigestHolder {
   }
 
   static void digestOfDigest(const DigestHolder& inDigest, DigestHolder& outDigest) {
```

Comment:
Please remove static from this fucntion.

Reply:
Addressed in https://github.com/arc-vmware/concord-bft/pull/3/commits/b7fc8a50c49f6f986000ec33fefe4d8e1580802d